README.md

```diff
@@ -199,7 +199,7 @@ Now you're all set.
 
 From now on, whenever you want to preview your application locally, just run:
 
-1. `npm run build`: This will run `next build` to build your Next.js app and `next-on-netlify` to prepare your Next.js app for compatibility with Netlify
+1. `npx next-on-netlify watch`: This will run `next build` to build your Next.js app and `next-on-netlify` to prepare your Next.js app for compatibility with Netlify. Any source code changes will trigger another build.
```

Comment:
how will this work with the auto-plugin?

Reply:
I think the plugin shouldn't need to do anything - it should always run only the build command.
The CLI should detect that the plugin is used, then when you run `ntl dev` it will run `next-on-netlify watch`